### PR TITLE
[ICP-12754] Fix for: "Qubino Flush On Off Thermostat 2 – Cannot set a heating temperature"

### PR DESCRIPTION
**Pull request by MGoralczykS**
When I changed the setpoint value I get this error 
`groovy.lang.GroovyRuntimeException: Ambiguous method overloading for method java.lang.Math#min.
Cannot resolve which method to invoke for [class java.math.BigDecimal, class java.lang.Integer] due to overlapping prototypes between:
[double, double]
[float, float] @line 281 (updateSetpoint)`

I cast those two values to double to fix this. 

@greens @SmartThingsCommunity/srpol-pe-team

### CODE REVIEW

File: devicetypes/smartthings/qubino-flush-thermostat.src/qubino-flush-thermostat.groovy [hunk 1]
```diff
@@ -278,7 +278,7 @@ def setCoolingSetpoint(setpoint) {
 
 def updateSetpoint(setpoint, setpointType) {
 	setpoint = temperatureScale == 'C' ? setpoint : fahrenheitToCelsius(setpoint)
-	setpoint = Math.max(Math.min(setpoint, maxSetpointTemperature), minSetpointTemperature)
+	setpoint = Math.max(Math.min(setpoint.doubleValue(), maxSetpointTemperature.doubleValue()), minSetpointTemperature)
```
Comment (KKlimczukS) [inline devicetypes/smartthings/qubino-flush-thermostat.src/qubino-flush-thermostat.groovy]: Can You also cast minSetpointTemperature to double ?

**Review by KKlimczukS — CHANGES_REQUESTED** [review]

File: devicetypes/smartthings/qubino-flush-thermostat.src/qubino-flush-thermostat.groovy (hunk 1 above)
Comment (MGoralczykS) [reply devicetypes/smartthings/qubino-flush-thermostat.src/qubino-flush-thermostat.groovy]: Changed

**Review by KKlimczukS — APPROVED** [review]

**Comment by greens** [comment]
what if `maxSetpointTemperature` and `minSetpointTemperature` just returned floats instead? I think you could just append a `.0` to all those values for a more elegant fix.

**Comment by KKlimczukS** [comment]
> what if `maxSetpointTemperature` and `minSetpointTemperature` just returned floats instead? I think you could just append a `.0` to all those values for a more elegant fix.

@greens I think that use case was checked by @MGoralczykS and it was working fine (with the code from this PR).
@MGoralczykS please confirm it.

**Comment by MGoralczykS** [comment]
@greens @KKlimczukS Everything is working correctly.

**Pull request merged**